src/test/librbd/fsx.cc

```diff
@@ -2029,9 +2023,10 @@ main(int argc, char **argv)
 				usage();
 			break;
 		case 'l':
-			maxfilelen = getnum(optarg, &endp);
-			if (maxfilelen <= 0)
+			int _num = getnum(optarg, &endp);
```

Comment:
@tchaikov why? It's in the same style as the rest of the file (which is taken from another project)